pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/SomaticResultUpload.java

```diff
@@ -39,69 +41,81 @@
 @JsonInclude(JsonInclude.Include.NON_NULL)
 public class SomaticResultUpload implements HasDdpInstanceId {
     @ColumnName(ESObjectConstants.SOMATIC_DOCUMENT_ID)
-    private final Long somaticDocumentId;
+    private Long somaticDocumentId;
```

Comment:
Hidden gem of the Elastic work.  We only reflect some of the work.  Had to make a default empty constructor of the class so deserialization would work in the ParticipantWrapper code base.